ghcup-0.0.7.yaml

```diff
@@ -4407,6 +4406,87 @@ ghcupDownloads:
             unknown_versioning:
               dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.10.1.0/cabal-install-3.10.1.0-armv7-linux-deb10.tar.xz
               dlHash: 79869dd72cb0a6f90f2e1e0e57af4ea6ee09b041550abc09252d1cf309b4f96e
+    3.10.2.0:
+      viPostInstall: "cabal run is currently partially broken on Windows, please see https://github.com/haskell/cabal/issues/9334"
+      viTags:
+        - Latest
+      viChangeLog: https://github.com/haskell/cabal/blob/master/release-notes/cabal-install-3.10.2.0.md
+      viArch:
+        A_64:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31020-64
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-alpine3_12.tar.xz
+              dlHash: b54e1cd235c47c62c03cdb9f6cf90e5fe8ae38c5e5befb9e21c8d1395f4bde05
+          Linux_Alpine:
+            unknown_versioning: *cabal-31020-64
+          Linux_CentOS:
+            unknown_versioning: &cabal-31020-64-centos7
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-centos7.tar.xz
+              dlHash: cfcdab399380dec7fedda55898bff975ac30b5d5d579433cbf8773b17c15f410
+          Linux_Debian:
+            ' ( == 9)': &cabal-31020-64-debian
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb9.tar.xz
+              dlHash: af5ce42114cf7720c37fee3238781df4c75bb74914c62e6a68833fb434e0cad7
+            ' ( == 10)':
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb10.tar.xz
+              dlHash: bdeb27c008b09c3b86f8a2768018d62a1aee02565304d123fda87ed432549418
+            ' ( >= 11)':
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb11.tar.xz
+              dlHash: 9ca5625c89e8fcada02edced5048c3a3db0254e2bef1eb792d549d633222b108
+            unknown_versioning: *cabal-31020-64-debian
```

Comment:
Yeah... not sure in this case actually. I don't think we have a consistent decision. There are two options in case we have at least 2 different bindists for a given distro, but can't figure out the version of the current distro:

1.  use the "older" bindist (debian 9 in this case)
    * can potentially cause issues with libraries like ncurses if the current "unknown" debian version is much higher
2.  use the newer bindist (debian 11 in this case)
    * can potentially cause issues with glibc if the current "unknown" debian version is much lower

For ubuntu, it seems we made the decision 2. I just realized that it's a corner case I haven't thought about very deeply.

Reply:
I do think we should default to the older bindist. That means we should also adjust `unknown_version` for Linux_Ubuntu x86, Linux_Mint x86 and Linux_Debian A_ARM64 .

Reply:
understood!